README.md

```diff
@@ -1,150 +1,235 @@
-
-## Information About The Files We Have Provided
-We have provided some file for you to make this tutorial easier to consume that I will talk about here.
-Firstly we have provided you with the basic directory structure for all the microservices.
-Then we have provided you with a pom.xml file that is used by maven to build your microservices and download all the required dependancies. In this file you can set what the application is named `<app.name>LibertyMicroServiceOne-1.0</app.name>`, the ports assigned to Liberty 
-```
-<testServerHttpPort>9090</testServerHttpPort> <testServerHttpsPort>9444</testServerHttpsPort>
-```
-the dependancies you require like microProfile 1.2
-```
-<dependency>
-    <groupId>org.eclipse.microprofile</groupId>
-    <artifactId>microprofile</artifactId>
-    <version>1.2</version>
-    <scope>provided</scope>
-    <type>pom</type>
-</dependency>
-```
-and the applicaiton server you are using so in our case open Liberty
+###About this module
+
+This module is a continuation of module 3 however, the finished module 3 code has been provided in this module 4 branch.
+
+This module will demonstrate MicroProfile Config. MP Config is a solution to externalise configuration from microservices, enabling users to separate configuration from code to enable microservices to be successfully run in different environments requiring different configurations. This helps to ensure the high portability of microservices.
+
+1.  Firstly, the static configuration properties in a static configuration properties file need to be set (this could be one of three pre-existing config files).
+2.  Following this, a custom configuration file needs to be created to enable dynamic configuration properties (Properties that can be updated without having to restart the server).
+3.  Finally, the new dynamic and static configuration properties need to be tested.
+
+If more help is needed in regards to any of the concepts and methods used in this module, please see the two interactive guides on http://OpenLiberty.io.guides : 
+1.  Using MicroProfile Config for static configuration injection [LINK]
+2.  Advancing the use of MicroProfile Configuration [LINK]
+
+## Before We Start
+
+Please ensure all of the prerequisites have been installed before continuing with these instructions. Clone down this repository
+https://github.com/OpenLiberty/tutorial-microprofile.git
+and navigate to this repository.
+Change branch to module 3 *tutorial-microprofile*
+*git checkout module4*
+
+### Enabling Static Configuration Properties
+This section will demonstrate how to set the HTTP and HTTPS port values, that the gateway server runs on, as examples of static configuration properties in a static configuration file. 
+1.  Once in the root directory, navigate into the following directory *microservice-gateway/src/main/java/liberty/config* and open the file called *bootstrap.properties*
```

Comment:
Same formatting issue here with it all going into one paragraph rather than one line